leakcanary-android-core/src/main/java/leakcanary/internal/activity/screen/LeakScreen.kt

```diff
@@ -198,12 +199,19 @@ private fun View.onLeakTraceSelected(analysis: HeapAnalysisSuccess, heapAnalysis
   private fun leakToString(
     leakTrace: LeakTrace,
     analysis: HeapAnalysisSuccess
-  ) = """$leakTrace
+  ): String {
+    var leakStr = """$leakTrace
 
 METADATA
 
 ${if (analysis.metadata.isNotEmpty()) analysis.metadata.map { "${it.key}: ${it.value}" }.joinToString(
-      "\n"
-  ) else ""}
+        "\n"
+    ) else ""}
 Analysis duration: ${analysis.analysisDurationMillis} ms"""
+
+    if (LeakCanary.config.wrapLeakTraceWhenSharing) {
+      leakStr = LeakTraceWordWrapper.wrap(leakStr, 80)
```

Comment:
nit: avoid 3 letter variable names (e.g. str), use full nouns instead. 

Here, instead of using a var you can return the original vs the wrapped one.